src/NUnit3TestAdapterInstallV2/source.extension.vsixmanifest

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
   <Metadata>
-    <Identity Id="NUnit.NUnit3TestAdapter" Version="4.0.0.0" Language="en-US" Publisher="Terje Sandstrom" />
+    <Identity Id="NUnit.NUnit3TestAdapter" Version="3.6.0.1" Language="en-US" Publisher="Terje Sandstrom" />
```

Comment:
I just need to do this to test the extension.  It is much easier than anything else.  I did the same earlier, you remember, with the 2.X adapter.  I could have dropped commiting this change, but then there are other changes in the same file.